bot/cogs/utils.py

```diff
@@ -250,8 +192,48 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
         for reaction in options:
             await message.add_reaction(reaction)
 
-    async def send_pep_zero(self, ctx: Context) -> None:
-        """Send information about PEP 0."""
+    # PEPs area
```

Comment:
this can use `# region: PEP` and later `# endregion` to be collapsible in some editors